CHANGELOG.md

```diff
@@ -85,6 +85,7 @@
 * [CHANGE] Ruler: changed ruler autoscaling policy, extended scale down period from 60s to 600s. #4786
 * [CHANGE] Update to v0.5.0 rollout-operator. #4893
 * [CHANGE] Backend: add `alertmanager_args` to `mimir-backend` when running in read-write deployment mode. Remove hardcoded `filesystem` alertmanager storage. This moves alertmanager's data-dir to `/data/alertmanager` by default. #4907 #4921
+* [CHANGE] Remove `-pdb` suffix from `PodDisruptionBudget` names. #5109
```

Comment:
Can this be a bit more explicit? According to [kubernetes docs](https://kubernetes.io/docs/tasks/run-application/configure-pdb/#arbitrary-controllers-and-selectors)

> The eviction API will disallow eviction of any pod covered by multiple PDBs, so most users will want to avoid overlapping selectors

So if someone doesn't prune resources generated by jsonnet, they will end up with stuck pods. Maybe you can mention something like 
```suggestion
* [CHANGE] Remove `-pdb` suffix from `PodDisruptionBudget` names. This will create new `PodDisruptionBudget` resources. Make sure to prune the old resources; otherwise, rollouts will be blocked. #5109
```

Reply:
Good idea, thanks.